common/libs/wallet/deterministic/ledger.ts

```diff
@@ -45,17 +45,29 @@ export class LedgerWallet extends HardwareWallet {
 
   public async signRawTransaction(t: EthTx): Promise<Buffer> {
     const txFields = getTransactionFields(t);
-    t.v = Buffer.from([t._chainId]);
+    t.v = toBuffer(t._chainId);
     t.r = toBuffer(0);
     t.s = toBuffer(0);
 
     try {
       const ethApp = await makeApp();
       const result = await ethApp.signTransaction(this.getPath(), t.serialize().toString('hex'));
 
+      let v = result.v;
```

Comment:
@hackmod 
Hey -- this code looks a little magical. Could we add more information here for the next person that sees this? An explanation comment?